types/theme-ui/theme-ui-tests.tsx

```diff
@@ -74,7 +74,10 @@ const workingThemeColorModes: Theme = {
 };
 
 // $ExpectError
-const incompleteThemeColorModes: Theme = { colors: { modes: { papaya: {
+const incompleteThemeColorModes: Theme = {
+    colors: {
+        modes: {
+            papaya: {
```

Comment:
This was prettier, should it be reverted?

Reply:
```diff
diff --git a/types/theme-ui/theme-ui-tests.tsx b/types/theme-ui/theme-ui-tests.tsx
index 8325993fe0..30ce1535e0 100644
--- a/types/theme-ui/theme-ui-tests.tsx
+++ b/types/theme-ui/theme-ui-tests.tsx
@@ -73,11 +73,8 @@ const workingThemeColorModes: Theme = {
     },
 };
 
-// $ExpectError
-const incompleteThemeColorModes: Theme = {
-    colors: {
-        modes: {
-            papaya: {
+// prettier-ignore
+const incompleteThemeColorModes: Theme = { colors: { modes: { papaya: { // $ExpectError
                 text: '#433',
             },
             dark: {
```

Reply:
Oh cool, and it turns out this change is what was causing the test to fail. That's fixed now too :)